### PR TITLE
Allow passing additional arguments to kubelet in hack/local-up-cluster.sh

### PR DESCRIPTION
**Pull request by dinvlad**
It would be nice to be able to pass additional arguments to kubelet started through `hack/local-up-cluster.sh`. Here I'm proposing to do it through `${KUBELET_ARGS}`.

### CODE REVIEW

**Comment by dinvlad** [comment]
I signed it!

File: hack/local-up-cluster.sh [hunk 1]
```diff
@@ -262,7 +262,7 @@ function start_kubelet {
         --hostname-override="127.0.0.1" \
         --address="127.0.0.1" \
         --api-servers="${API_HOST}:${API_PORT}" \
-        --port="$KUBELET_PORT" >"${KUBELET_LOG}" 2>&1 &
+        --port="$KUBELET_PORT" ${KUBELET_ARGS} >"${KUBELET_LOG}" 2>&1 &
```
Comment (lavalamp) [inline hack/local-up-cluster.sh]: I recommend initializing at the top in case it is unset, with the other variables.

Comment (dinvlad) [inline hack/local-up-cluster.sh]: Sounds good. I guess that is only cosmetic in this case (makes it a bit more obvious what parameters are available, and a good programming practice nevertheless).

Comment (dinvlad) [inline hack/local-up-cluster.sh]: .. which suggests we should probably add `set -o nounset` to this script, as done for others under `hack/`

Comment (dinvlad) [inline hack/local-up-cluster.sh]: Good call.There's even a check for unbound variable that fails otherwise.

**Comment by wojtek-t** [comment]
LGTM

**Comment by wojtek-t** [comment]
@k8s-bot ok to test

**Pull request merged**